app/scripts/lib/experiments/grouping-rules/signup-password-confirm.js

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define((require, exports, module) => {
+  'use strict';
+
+  const BaseGroupingRule = require('./base');
+
+  module.exports = class SignupPasswordConfirmGroupingRule extends BaseGroupingRule {
+    constructor () {
+      super();
+      this.name = 'signupPasswordConfirm';
+    }
+
+    choose (subject = {}) {
+      const sampleRate = SignupPasswordConfirmGroupingRule.sampleRate(subject.env);
+
+      return !! subject.uniqueUserId && this.bernoulliTrial(sampleRate, subject.uniqueUserId);
```

Comment:
I saw mention of using bernoulli and uniform together, but in this case, I didn't see why that would be useful. If we say 50% of people get the experiment, and of those people, half are put in treatment, and half in control, well, that's just 25% of users having the experiment enabled, right? Since the 'control' element of this experiment is to just not change anything, then anyone not selected is 'control'.

I do see how such a thing would be useful in an experiment where there are more than 2 groups, or where the 2 groups are both different from those not in the experiment.